base/types/gfsperrors/error.go

```diff
@@ -43,6 +46,26 @@ func init() {
 	})
 }
 
+func GetGfSpErr(err error) *GfSpError {
+	if err == nil {
+		return nil
+	}
+	if gfspErr, ok := err.(*GfSpError); ok {
+		return gfspErr
+	}
+	i := strings.Index(err.Error(), "desc = ")
+	if len(err.Error())-1 < i+6 {
+		return nil
+	}
+	errInfo := err.Error()[i+6:]
```

Comment:
Does `[i+6:]` fit all cases?

Reply:
The result after the rpc call is "rpc error: code = Unknown desc = xxx" We only focus on xxx